django/contrib/admin/static/admin/css/widgets.css

```diff
@@ -90,26 +90,32 @@
     display: block;
     text-indent: -3000px;
     overflow: hidden;
+    cursor: default;
+    opacity: 0.3;
```

Comment:
Righto. https://github.com/ryangallen/django/commit/080cf1ef3fd8e80cb7938f35abfe0a4d95c0ea83 should fix that